enum MsgType
{
    C2Gate_MsgNull				 = 0;
    C2Gate_MsgLoginReq			 = 1;
    C2Gate_MsgCreateAcountReq	 = 2;
	C2Gate_MsgLoginMacReq		 = 3;
	C2Gate_END					 = 998;

	G2C_Begin					 = 999;

	G2C_LoginMacACK				 = 1000;
}


message MsgHead
{
  required MsgType Type = 1;
  required int32 msgSize = 2;	
}
